scripts/ai2-internal/run_with_beaker.py

```diff
@@ -25,7 +25,13 @@ def main(param_file: str, args: argparse.Namespace):
     overrides = ""
 
     # Reads params and sets environment.
-    params = Params.from_file(param_file, overrides)
+    ext_vars = {}
+
+    for var in args.env:
```

Comment:
I think this is a massive breaking change, I don't think we should do this. Is there something wrong with pulling in the environment variables inside beaker as is possible already?

Reply:
The problem @joelgrus ran into was that `Params.from_file` was failing because it didn't have access to the parameters that were passed in via `--env`.

Reply:
the problem this was trying to solve is that the run_with_beaker script is parsing the params file to inject extra env vars into beaker. in particular, if your config contains an env variable, your local machine needs that env variable in order for this parsing code not to crash.

then you *also* need to supply those env variables as command line arguments

Reply:
so I would have to do something like 

```
MY_FILE=/data/my_file.txt beaker experiment run --env=MY_FILE=/data/my_file.txt ...
```

which seems silly

Reply:
The only reason we load the params file here is so that it appears in the beaker UI - I see that this is problematic if you have environment variables set locally (because we would pick them up and make the substitutions when we read it in here), but I don't think this is the fix for it.

Reply:
an alternative solution would leave `Params` alone and inject all of the `--env` variables into your local environment -- I thought my solution was cleaner and potentially more broadly useful

Reply:
Sorry ignore this. I misread your changes.